pages/home.md

```diff
@@ -8,88 +8,162 @@ permalink: /
+      Using Web standards to
+      let people control their data, and choose the
+      applications and services to use with it.
       </p>
+      <div class="buttons is-centered">
+        <!--
+        <a href="{{site.baseurl}}/about/">
+          <button class="button is-primary">About Solid</button>
+        </a>
+        -->
+        <a href="{{site.baseurl}}/users/">
+          <button class="button is-primary">Get a Pod</button>        
+        </a>
+        <a href="{{site.baseurl}}/developers/tutorials/getting-started">
+          <button class="button is-primary">Developer Tutorial</button>
+        </a>
+      </div>
     </div>
   </div>
 </section>
 
 <div id="landing-content">
-  {%
-    include choose-your-own-adventure.html
-      heading="Solid is great for…"
-      link1="/use-solid"
-      adventure1="Everyone"
-      link2="/for-developers"
-      adventure2="Developers"
-      link3="/for-enterprises"
-      adventure3="Enterprises"
-  %}
 
-  <div id="tour">
-    <section class="section">
-      <div class="container">
+  <section id="overview" class="section">
+
+    <div class="container">
+
+        <p class="subtitle has-text-centered is-3">
+          Solid lets people store their data in decentralized data stores 
+          called <b>pods</b>.
+        </p> 
+
         <div class="columns">
-          <div class="column is-hidden-touch is-offset-1 is-4">
-            <figure class="image">
-              <img src="{{site.baseurl}}/assets/img/single-sign-on.svg" alt="[]" />
-            </figure>
-          </div>
-          <div class="column is-offset-1 is-5">
-            <h2 class="title">The last login you'll ever need</h2>
-            <p class="content is-large">
-              Solid provides for the first time a <b>single global logon system</b>, so that when you log into any web site, instead of having  to log in with the usual 'f' and 'g', etc, blue buttons, and then be tracked by Facebook, Google, or some other large social network, instead you can log in with any Solid provider you trust, and that won't track you.
+          <div class="column is-half-desktop is-offset-one-quarter-desktop">
+          
+            <p class="content has-text-centered is-large">
+              Pods are like secure personal web servers for data.
+              All data in a pod is accessible
+              accesible via the 
+              <a href="https://solid.github.io/specification/">Solid Protocol</a>.
+              When data is stored in someone's 
+              pod, they control who and what can access it.
             </p>
+            
+            <p class="content has-text-centered is-large">
+            Solid is led by the inventor of the Web, 
+            <b><a href="{{site.baseurl}}/team/">Sir Tim Berners-Lee</a></b>, 
+            to help realize his vision 
```

Comment:
Just realised: we've been using British English throughout the site so far. I have no idea how that's perceived by native speakers, so I'm just flagging it.

Reply:
Good point. I'd say it's more important to be consistent, so I'd edit this to be in line with whatever the rest of the site currently uses (and then open an issue to get **_formal agreement_** (if there isn't already) on UK vs US English overall).

Reply:
Agree with @pmcb55. For now I've corrected that to use british english for consistency.